lib/chat_app_web/live/single_room_live.ex

```diff
@@ -7,23 +7,62 @@ defmodule ChatAppWeb.SingleRoomLive do
   @impl true
   def mount(%{"id" => room_id}, _session, socket) do
     topic = "room:" <> room_id
+
     if connected?(socket) do
-        Logger.info("Liveview's second mount() call")
-        LiveviewMonitor.monitor(self(), __MODULE__, %{id: socket.id, room_id: room_id})
-        PubSub.subscribe(ChatApp.PubSub, topic)
-        broadcasted_message = {:add_messages, ChatApp.Message.new("anon joined the chat.", "", room_id)}
-        PubSub.broadcast(ChatApp.PubSub, topic, broadcasted_message)
+      Logger.info("Liveview's second mount() call")
+      LiveviewMonitor.monitor(self(), __MODULE__, %{id: socket.id, room_id: room_id})
+      PubSub.subscribe(ChatApp.PubSub, topic)
+
+      message = %{
+        "content" => "anon joined the chat",
+        "sender" => "",
```

Comment:
we need to identify the joining/leaving user.

Reply:
That will be done in the home_live/ETS integration task.